content/docs/09.5-security/10-hardened-infrastructure.md

```diff
@@ -0,0 +1,153 @@
+
+Palette's OS hardening utility performs the following tasks:
+
+<br />
+
+- Applies the latest available security updates.
+
+
+- Hardens SSH server parameters, network parameters (sysctl), and system files by ensuring proper file permissions are set.
+
+
+- Removes legacy services and graphical user interface (GUI) packages.
+
+Palette allows you to set up OS patching policies. You can patch the base OS when you deploy the cluster.
+
+<br />
+
+## Hardened Containers
+
+Container images for various application services are built using distroless images, which have significantly fewer packages and improve security by reducing attack surface.
+
+All container images are scanned for vulnerability using Palo Alto Networks Prisma Cloud (Twistlock) Defender before being published to a repository or deployed to the SaaS platform.
+
+<br />
+
+## Hardened Kubernetes
+
+Spectro Cloud has a fully automated Kubernetes verification system that adds the newest patch version of Kubernetes to its public repository. All Kubernetes packs are hardened according to CIS Benchmarks.
+
+A major Kubernetes version generally runs more rigorous compatibility tests and is available after 30 days from its release.
+
+Kubernetes run-time security support is achieved through a variety of add-on packages, such as Sysdig Falco and Twistlock.
+
+Kubernetes cluster authentication can be optionally integrated with Kubelogin using OpenID Connect (OIDC) authentication and authorization against an external Identity Provider (IDP).
+
+You can set a schedule to run Kubernetes conformance and compliance tests using kube-bench, kube-hunter, and Sonobuoy. These tests ensure tenant clusters are secure, compliant, and up to date.
+
+<br />
+
+## Secure Keys 
+
+The following encryption keys are unique and generated for each installation:
+
+<br />
+
+- **Root Key**: Encrypts the tenant-specific encryption key.
+
+
+- **JSON web token (JWT) signature key**: Used to sign the JWT token.
+
+
+- **Hash Salt**: Used to hash the user password and email ID.
+
+
+- **Tenant key**: A 64-bit cryptographically secure tenant key encrypts tenant data. A unique tenant key is generated for each tenant and encrypts the following: <br /><br />
+
+    
+    - User account name and email ID.
+    
+    
+    - Tenant's cloud credentials.
+
+    
+    - Tenant's private registry password.
+
+    
+    - Tenant's GitHub access token.
+
+
+In self-managed deployments, keys are generated during installation and stored as secrets in the management cluster’s etcd key-value store.
+
+
+<br />
+
+## Data Encryption
+
+In SaaS deployments, backup snapshots of MongoDB cluster storage volumes are encrypted to secure cluster data on disk. In tenant clusters, Palette ensures security for data at rest and data in transit. Refer to [Tenant Cluster Security](/security/saas-operation#tenantclustersecurity) to learn how Palette applies data security.
```

Comment:
This is because the content was a bit too much "re-shuffled": the platform itself (Palette) encrypts at rest data, but not the target clusters. Please check the comment I'll do after the review in the PR.